core/src/main/java/com/linecorp/armeria/server/healthcheck/HealthCheckUpdateHandler.java

```diff
@@ -28,6 +28,14 @@
  */
 @FunctionalInterface
 public interface HealthCheckUpdateHandler {
+
+    /**
+     * Returns the default {@link HealthCheckUpdateHandler} implementation.
+     */
+    static HealthCheckUpdateHandler ofDefault() {
+        return DefaultHealthCheckUpdateHandler.INSTANCE;
```

Comment:
Not related to this PR but how about making this as a singleton enum?

Reply:
I missed it. Nice idea, is this what you intended?

```java
enum DefaultHealthCheckUpdateHandler implements HealthCheckUpdateHandler {

    INSTANCE;

    private static final ObjectMapper mapper = new ObjectMapper();

    @Override
    public CompletionStage<HealthCheckUpdateResult> handle(ServiceRequestContext ctx, HttpRequest req) throws Exception {
        ...
    }
}
```

Reply:
Yes.